.github/workflows/benchmark-command.yml

```diff
@@ -25,12 +27,17 @@ on:
           - high
           - good
           - medium
+          - dev
+          - stable
       benchmark:
         description: 'Benchmark to run'
         required: true
         options:
           - full
           - test
+          - small
+          - timeseries_test
+          - none (for AutoMM)
```

Comment:
`none (for AutoMM)` may not error out as string here but it doesn't look synthetically fit here.

Reply:
Might have missed this during cleanup, will change it